css-borders-4/Overview.bs

```diff
@@ -617,16 +617,29 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either 'none' or a list,
-	each item a pair of offsets (horizontal and vertical) from the element‘s box
-Animation type: by computed value
+Computed value: list, each item either 'none' or a pair of offsets
+  (horizontal and vertical) from the element‘s box
+Animation type: by computed value,
+    letting ''box-shadow-offset/none'' create a blank shadow
+    (''transparent 0 0 0 0'') when interpolated with non-'none' values
```

Comment:
I don't think that works, the computed value of a property can affect another, but if the computed value at the midpoint between `none` and `10px 10px` is `5px 5px`, then `5px 5px` shouldn't by itself force `box-shadow-color` to get a `transparent`.

As I said, I think handling `transparent` in the shorthand expansion will suffice to keep the old behavior.

Reply:
Ok, I've moved the handling of `transparent` to the shorthand expansion now.